internal/k3d/k3d.go

```diff
@@ -302,6 +308,23 @@ func getK3sImage(kubernetesVersion string) (string, error) {
 	return fmt.Sprintf("rancher/k3s:v%s-k3s1", kubernetesVersion), nil
 }
 
+// getK3sImageForV5 only takes the major version of kubernetes and transform it to a k3s image tag from the channelserver (See https://update.k3s.io/v1-release/channels).
+func getK3sImageForV5(kubernetesVersion string) (string, error) {
```

Comment:
comments in this function scope are a bit noisy. can you maybe reduce them to minimal needed and shorten a bit?